x-pack/packages/kbn-entities-schema/src/schema/entity.ts

```diff
@@ -15,6 +15,7 @@ export const entitySchema = z.intersection(
       indexPatterns: arrayOfStringsSchema,
       identityFields: arrayOfStringsSchema,
       metric: z.record(z.string(), z.number()),
+      spaceId: z.string(),
```

Comment:
should this be `z.optional(z.string())` ?

Reply:
I'm going to leave that as a required field since we will fill it with `default` by... uh... *default*.